roles/common/meta/main.yml

```diff
@@ -25,3 +25,4 @@ dependencies:
     when: logging.enabled|default(True)|bool
     service: common
     logdata: "{{ common.logs }}"
+  - role: serverspec
```

Comment:
so, we should drop this as a dep of the `common` role. and we can add it directly under `site.yml` in the beginning, using a `when: serverspec.enabled|default("True")|bool`